deploy/file_list

```diff
@@ -1,8 +1,10 @@
-Assets/Plugins/Android/SoomlaAndroidCore.jar
```

Comment:
@EugeneButusov ok lets revert this to what it was :) Define each file separately.